es-core/src/AudioManager.cpp

```diff
@@ -120,9 +97,9 @@ void AudioManager::registerSound(std::shared_ptr<Sound> & sound)
 void AudioManager::unregisterSound(std::shared_ptr<Sound> & sound)
 {
 	getInstance();
-	for(unsigned int i = 0; i < sSoundVector.size(); i++)
+	for (unsigned int i = 0; i < sSoundVector.size(); i++)
```

Comment:
Style changes of existing code should be done separately imho.

Reply:
The [original](https://github.com/RetroPie/EmulationStation/blob/master/es-core/src/AudioManager.cpp) was not consistent so I stuck with the spaces in the conditionals.